plugins/modem-manager/fu-mm-utils.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2019 Aleksander Morgado <aleksander@aleksander.es>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include <gio/gio.h>
+#include "fu-mm-utils.h"
+
+gboolean
+fu_mm_utils_get_udev_port_info (GUdevDevice  *device,
+				gchar	    **out_device_sysfs_path,
+				gint	     *out_port_ifnum,
+				GError	    **error)
+{
+	g_autoptr(GUdevDevice) parent = NULL;
+	g_autofree gchar *device_sysfs_path = NULL;
```

Comment:
convention in the codebase is that the g_auto* stuff goes last.

Reply:
done